src/main/java/de/hysky/skyblocker/skyblock/item/tooltip/adders/BitsHelper.java

```diff
@@ -0,0 +1,448 @@
+
+        if (categories.containsKey(categoryName)) {
+            LOGGER.debug(LOGS_PREFIX+"Key matched for: {}", categoryName);
+            Map<String, Long> results = new HashMap<>();
+            Map<String, Integer> category = categories.get(categoryName);
+
+            for (Map.Entry<String, Integer> entry : category.entrySet()) {
+                String itemID = entry.getKey();
+                Integer itemBitsPrice = entry.getValue();
+                double itemCost = getPrice(itemID);
+                LOGGER.debug(LOGS_PREFIX+"Line processed: {} item, {} price in bits, {} price in coins", itemID, itemBitsPrice, itemCost);
+                long coinsPerBit = Math.round(itemCost / itemBitsPrice);
+                results.put(itemID, coinsPerBit);
+            }
+            return results;
+        } else if (categoryName.contains("Fuel Blocks")) {
+            LOGGER.debug(LOGS_PREFIX+"Fuel Blocks code triggered"); // so that thing gets cheaper if player has blaze slayer 9
+            String itemID = "INFERNO_FUEL_BLOCK";   // but I don't know if only 1x offer of 64x offer gets discount too
+            int[] itemBitsPrice = {75, 3600};   // if only 1x gets discount then it doesn't matter as x64 would be ALWAYS better even with it
+            double itemCost = getPrice(itemID); // TLDR: need blaze slayer 9 player to show their prices
+            long coinsPerBit = (long) (Math.max(itemCost / itemBitsPrice[0], itemCost * 64 / itemBitsPrice[1]));
+            return Collections.singletonMap(itemID, coinsPerBit);
+        } else {
+            LOGGER.warn(LOGS_PREFIX+"Can't recognize category {} from bits shop! Consider reporting this to our discord!", categoryName);
+        }
+        return Collections.emptyMap();
+    }
+
+
+    /**
+     * List of items that sell well. No, it is not automated, but that stuff is unlikely to change unless some update
+     * Definition of "good selling item": 300+ daily sales, 12+ hourly sales
+     * I would personally also demand good price per item as filling up all ah slots with catalyst would suck
+     * But newer players would find a good use of it so ima keep my preferences away
+     * Actual on 14.08.2024
+     */
+    private final List<String> sellingItems = Arrays.asList(
+            "KAT_FLOWER",
+            "KAT_BOUQUET",
+            "HEAT_CORE",
+            "ULTIMATE_CARROT_CANDY_UPGRADE",
+            "TALISMAN_ENRICHMENT_SWAPPER",
+            "GOD_POTION_2",
+            "KISMET_FEATHER",
+            "MATRIARCH_PARFUM",
+            "AUTOPET_RULES_2"   // it barely made it here
+    );
+
+    /**
+     * SKYBLOCK_ID, Bits Price
+     * Yes I just hardcoded it. Why work smart when you can work hard?
+     * Those are only from submenus (categories) as normal item is easy to parse
+     * It probably could've been extracted from neu repo or something
+     * Actual on 14.08.2024
+     */
+    private final Map<String, Integer> catKat = Util.make(new HashMap<>(), map -> {
```

Comment:
Use an Object2IntMap instead for these maps

Also these constants should follow the naming conventions